mlflow/tracing/display/display_handler.py

```diff
@@ -4,7 +4,7 @@
 
 from mlflow.entities import Trace
 
-MAX_TRACES_TO_DISPLAY = 100
+MAX_TRACES_TO_DISPLAY = 10
```

Comment:
Can we make it an environment variable we can change? Or add some helper function

Reply:
+1 to env var!